deploy/openshift/README.md

```diff
@@ -2,7 +2,7 @@
 Deployment depends strongly on Openshift installation which is described in this [documentation](https://docs.openshift.com/container-platform/4.1/installing/installing_aws/installing-aws-customizations.html)
 
 Prerequisities that have to be fulfilled in order to dpeloy Contrail with operator on Openshift:
-* openshift-install binary ([download](https://cloud.redhat.com/openshift/install))
+* openshift-install binary (>=4.4.8) ([download](https://cloud.redhat.com/openshift/install))
```

Comment:
Can you also add, in line 36 of README.md, information to not use the -rhel tags, just plain master.1245?
E.g. : 
Under *CONTRAIL_VERSION* field enter proper Contrail container build tag, available in the hub.juniper.net/contrail-nightly registry.
Do not specify the `-rhel` suffix in the build tag, it will be appended automatically where needed.

Reply:
Done